src/app/AttributeCache.cpp

```diff
@@ -29,6 +29,13 @@ CHIP_ERROR AttributeCache::UpdateCache(const ConcreteDataAttributePath & aPath,
     AttributeState state;
     System::PacketBufferHandle handle;
     System::PacketBufferTLVWriter writer;
+    bool findNewEndpoint = false;
+    ConcreteClusterPath currentClusterPath(aPath.mEndpointId, aPath.mClusterId);
+
+    if (mCache.find(aPath.mEndpointId) == mCache.end())
+    {
+        findNewEndpoint = true;
```

Comment:
i prefer keeping this boolean with renaming endpointIsNew, since the original behavior is to add endpoint when apData is not there and statusIB is there, and don't plan to modify this behavior